hvm/installer/install.go

```diff
@@ -0,0 +1,49 @@
+package installer
+
+import (
+	"bytes"
+	"context"
+	"io/fs"
+	"io/ioutil"
+	"os"
+
+	"github.com/chanzuckerberg/go-misc/errors"
+	"github.com/chanzuckerberg/happy/shared/githubconnector"
+	"github.com/codeclysm/extract"
+)
+
+func InstallPackage(org, project, version, os, arch, binPath string) error {
+
+	client := githubconnector.NewConnectorClient()
+
+	downloaded, err := client.DownloadPackage(org, project, version, os, arch, "/tmp")
+	if err != nil {
+		return err
+	}
+
+	err = doInstall(downloaded, binPath)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func doInstall(sourcePackagePath, binPath string) error {
```

Comment:
```suggestion
func doInstall(ctx context.Context, sourcePackagePath, binPath string) error {
```

thread through the context rather than using context.TODO. Cobra will have a context object and you can pass it in using `cmd.Context()`. It probably doesn't help too much, but will be more important if this ever uses goroutines in the future.

Reply:
> Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx:

https://pkg.go.dev/context